test-case-reporting/test/testing_utils.ts

```diff
@@ -31,8 +31,8 @@ export async function truncateAll(db: Database): Promise<void> {
 /**
  * Fills the database with builds, test cases, jobs, and test runs.
  * Stores 3 test cases.
- * Stores 2 builds: the first one has 5 jobs and
- * 15 test runs (one per (job, test case) pair).
+ * Stores 2 builds: the first one has 6 jobs and
+ * 18 test runs (one per (job, test case) pair).
```

Comment:
Given that you've had to change this already within a single PR, it seems like it may be too volatile to trust that it will get updated whenever the test data is updated. I'd say you can remove these specific #s

Reply:
Done.